packages/common/src/SVGWidget.ts

```diff
@@ -301,8 +301,10 @@ export class SVGWidget extends Widget {
                 if ((transX < 0 || transY < 0) && this._boundingBox) {
                     transX = transX < 0 ? 0 : transX;
                     transY = transY < 0 ? 0 : transY;
-                    this._parentRelativeDiv.style("overflow", "scroll");
-                    this._placeholderElement.attr("width", this._boundingBox.width);
+                    this._parentRelativeDiv.style("overflow-y", "scroll");
+                    this._parentRelativeDiv.style("overflow-x", "hidden");
+                    this._parentRelativeDiv.style("width", this._boundingBox.width + 8 + getScrollbarWidth() + "px");
+                    this._placeholderElement.attr("width", this._boundingBox.width + 8 + getScrollbarWidth());
```

Comment:
![image](https://user-images.githubusercontent.com/966863/73062813-2cbf3880-3e95-11ea-977d-c9263072783e.png)

I suspect that fix should be here:  https://github.com/hpcc-systems/Visualization/blob/master/packages/layout/src/Border2.ts#L132 ?

Reply:
I've not been able to recreate that issue outside of playground. I'm working on it though.